txpipe/random_cats.py

```diff
@@ -93,8 +98,10 @@ def run(self):
         x = 10.0 ** (0.4 * (Mstar - depth))
         density = phi_star * scipy.special.gammaincc(alpha15, x)
 
-        # Pixel geometry - area in arcmin^2
-        pix_area = scheme.pixel_area(degrees=True) * 60.0 * 60.0
+        # Pixel geometry - area of a single pixel in arcmin^2
+        full_pix_area = scheme.pixel_area(degrees=True) * 60.0 * 60.0
+        pix_area = frac*full_pix_area
```

Comment:
Maybe add a TODO here to use the higher resolution healsparse map to populate the randoms? Or does that not matter once you're at high enough res anyway?

Reply:
yea good idea. Our mask should be at high enough resolution that it wont matter, but it would be nice to have. Would require using the high-res "supreme" map as input rather than the mask

Reply:
as for this PR, this just means adding a comment in the code to say this is a TO DO Item?

Reply:
A TODO is fine for now. Let's see if we need this.

Reply:
ok I've added that